dwave_networkx/generators/zephyr.py

```diff
@@ -47,22 +48,38 @@ def zephyr_graph(m, t=4, create_using=None, node_list=None, edge_list=None,
     create_using : Graph, optional (default None)
         If provided, this graph is cleared of nodes and edges and filled
         with the new graph. Usually used to set the type of the graph.
-    node_list : iterable, optional (default None)
-        Iterable of nodes in the graph. If None, calculated from ``m``.
-        Note that this list is used to remove nodes, so only specified nodes
-        that belong to the base node set (described in the ``coordinates``
-        parameter) are added.
-    edge_list : iterable, optional (default None)
-        Iterable of edges in the graph. If None, edges are generated as
-        described below. The nodes in each edge must be labeled according to the
-        ``coordinates`` parameter.
+    node_list : iterable (optional, default None)
+        Iterable of nodes in the graph. If None, calculated from (``m``, ``t``)
```

Comment:
I think either ``None`` or "not specified" (preferable). Also below

Reply:
Changed 'If None' to 'If not specified' everywhere.